packages/esm-patient-chart-app/src/view-components/side-menu.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from "react";
+import { SideNav } from "carbon-components-react/es/components/UIShell";
+import { SideNavProps } from "carbon-components-react";
```

Comment:
Import directly from the respective module

Reply:
https://github.com/openmrs/openmrs-esm-patient-chart/pull/217/commits/52655ef43e8cc0d771874e644010e91bd78609bf this is the right import?

Reply:
I guess so! Thanks!